transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollServerChannel.java

```diff
@@ -102,7 +102,6 @@ void epollInReady() {
                 return;
             }
             final EpollRecvByteAllocatorHandle allocHandle = recvBufAllocHandle();
-            allocHandle.edgeTriggered(isFlagSet(Native.EPOLLET));
```

Comment:
is the EPOLLET still used in other places?

Reply:
yep it is...